docs/directives.md

```diff
@@ -84,20 +84,24 @@ class Foo {
 
 Besides declaring the usage of directives, you also have to register the runtime part of the used directives.
 
-> Be aware that TypeGraphQL doesn't have any special way for implementing schema directives. You should use some [3rd party libraries](https://www.apollographql.com/docs/graphql-tools/schema-directives/#implementing-schema-directives) depending on the tool set you use in your project, e.g. `graphql-tools` or `ApolloServer`.
+> Be aware that TypeGraphQL doesn't have any special way for implementing schema directives. You should use some [3rd party libraries](https://the-guild.dev/graphql/tools/docs/schema-directives#implementing-schema-directives) depending on the tool set you use in your project, e.g. `@graphql-tools/*` or `ApolloServer`.
 
-Here is an example using the [`graphql-tools`](https://github.com/apollographql/graphql-tools):
+Here is an example using the [`@graphql-tools/*`](https://the-guild.dev/graphql/tools):
 
-```typescript
-import { SchemaDirectiveVisitor } from "graphql-tools";
+```ts
+import { mergeSchemas } from "@graphql-tools/schema";
 
-// build the schema as always
-const schema = buildSchemaSync({
+// Build TypeGraphQL executable schema
+const schemaSimple = await buildSchema({
   resolvers: [SampleResolver],
 });
 
-// register the used directives implementations
-SchemaDirectiveVisitor.visitSchemaDirectives(schema, {
-  sample: SampleDirective,
+// Merge schema with sample directive type definitions
+const schemaMerged = mergeSchemas({
+  schemas: [schemaSimple],
+  typeDefs: [sampleDirective.typeDefs],
 });
```

Comment:
I don't think it's needed. TypeGraphQL requires providing `GraphQLDirective` instanced to `directives` build option. Then you have the directive transformer for implementation (see `testDirectiveTransformer` in tests).